prody/proteins/interactions.py

```diff
@@ -3786,3 +4047,192 @@ def parseLigandInteractions(self, filename):
         
         return data
 
+
+    def getInteractionTypes(self):
+        """Show which interaction types were detected for ligands."""
+        
+        interactions = self._interactions_traj 
+        unique_keywords = set()
+
+        for sublist in interactions:
+            for sublist_item in sublist:
+                keyword = sublist_item[0]
+                unique_keywords.add(keyword)
+
+        unique_keywords_list = list(unique_keywords)
+        LOGGER.info("Interaction types: {0}".format(unique_keywords_list))
+        
+        return unique_keywords_list
+
+
+    def getLigandsNames(self):
+        """Show which ligands are in a system."""
+        
+        interactions = self._interactions_traj 
+        ligands = set()
+
+        for sublist in interactions:
+            for sublist_item in sublist:
+                keyword = sublist_item[4]
+                ligands.add(keyword)
+
+        ligands_list = list(ligands)
+        
+        return ligands_list
+
+
+    def getFrequentInteractors(self, **kwargs):
+        """Provide a dictonary with residues involved in the interaction with ligand
```

Comment:
returns instead of provide. Provide sounds like an instruction for users

Also, why does this start with get not calc?

Reply:
Done. Now it will be calcFrequentInteractors()